tensorboard/program.py

```diff
@@ -77,19 +77,19 @@ class TensorBoard(object):
 
     def __init__(
         self,
-        assets_zip_provider,
         plugins=None,
+        assets_zip_provider=None,
         server_class=None,
         subcommands=None,
     ):
         """Creates new instance.
 
         Args:
-          assets_zip_provider: A function that provides a zip file containing assets to
-            the application.
           plugins: A list of TensorBoard plugins to load, as TBPlugin classes or
             TBLoader instances or classes. If not specified, defaults to first-party
             plugins.
+          assets_zip_provider: A function that provides a zip file containing assets to
+            the application.
```

Comment:
Maybe mention what happens if this is omitted/None?

Reply:
Done.